packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -529,6 +550,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
             content,
             fluid: true,
             styles: styles.triggerButton,
+            ...(label && { 'aria-labelledby': labelProps['id'] }),
```

Comment:
I guess it'd be difficult to separate this to an accessibility behavior at this point, but can we compute the `labelProps` (using Downshift function) in separate method (and preferably file) and spread the result here; 

the point I'm trying to make is that we should decouple these concerns as much as we can. the area where we apply props should work as an aggregator:

```typescript
{Button.create(triggerButton, {
  defaultProps: {
    ...stardustProps,
    ...accessibilityProps, // we should not care it's coming from Downshift 
                           // so that we can easily refactor if needed
    ...restProps,
  },
  // ...
})

Reply:
added them to a separate method.